src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -275,6 +306,18 @@ export default class NewDataset extends Vue {
       ? this.collectedMetadata.filesInfo
       : null;
 
+    const framesAsAxes: { [dim: string]: number } = {};
+    Object.entries(this.strides).forEach(([key, value]) => {
+      if (value) {
+        if (typeof value === "string") {
```

Comment:
I think this conversion should be done before being stored, what do you think?

Reply:
It would be better so that we only need to support `null` and `number`
We need a setter for that because `v-text-field` edits `strides`
But I didn't manage to create a setter for `strides`, because it is an object